src/openforms/registrations/contrib/objects_api/config.py

```diff
@@ -56,7 +56,10 @@ class ObjecttypeVariableMappingSerializer(serializers.Serializer):
         help_text=_(
             "Representation of the JSON target location as a list of string segments."
         ),
+        required=False,
     )
+    # specific options according to the variable key/type
+    options = JSONFieldWithSchema(default={}, required=False)
```

Comment:
```suggestion
    options = JSONFieldWithSchema(default=dict, required=False)
```
I'm always scared when I see a mutable default argument :)